packages/bpk-component-rating/src/common-types.js

```diff
@@ -18,7 +18,7 @@
 
 /* @flow */
 
-export const RATING_SIZES = { base: 'base', large: 'lg' };
+export const RATING_SIZES = { base: 'base', large: 'large' };
```

Comment:
What's the reason behind this? This would make this change breaking and might cause more issues for the consumer

Reply:
Hi Ollie, thanks for your review. It due to there is conflict between [propTypes](https://github.com/Skyscanner/backpack/blob/ced06446d5dd6be464b17bee92876848d26bfb34/packages/bpk-component-rating/src/BpkRating.js#L157) and  [typescript Props](https://github.com/Skyscanner/backpack/blob/ced06446d5dd6be464b17bee92876848d26bfb34/packages/bpk-component-rating/src/BpkRating.js#L51). It would throw waring as bellow.
![image](https://user-images.githubusercontent.com/33623220/170968018-cf2dffe9-cf35-46d5-a806-33c356a981cb.png)

Reply:
Since it would change the value what consumer pass, it change to `Breaking` in `UNRELEASED.md`